src/app/game.ts

```diff
@@ -203,40 +207,45 @@ export class Game {
     return this.gameMessages
   }
 
-  addGameMessage(message: string) {
+  addGameMessage(message: GameMessage) {
     this.gameMessages.push(message)
   }
 
   handleGoldmineWasBuilt(goldmine: Goldmine) {
     const gold = goldmine.getProduction()
 
-    this.addGameMessage(
-      `The miners did their best. You have got ${gold} gold ${
+    this.addGameMessage({
+      message: `The miners did their best. You have got ${gold} gold ${
         gold === 1 ? 'bar' : 'bars'
-      }.`
-    )
+      }.`,
+      type: MessageType.SUCCESS,
```

Comment:
I wonder what message should be here, i think the most messages should be info, success I would make only something like winning the battle or in the future when we add functionality to add by random resources, then we can show it as a success.